gulpfile.js

```diff
@@ -33,4 +33,17 @@ elixir(function(mix) {
         'socialmediashare.js'
     ], 'public/js/scripts.js', 'public/js');
 
+    /**
+     * Compile sass files to css
+     */
+    mix.sass('main.scss')
+       .sass('audioplayer.scss');
```

Comment:
I still need some clarification here concerning the sass files, since we are not writing in sass, do we still need this block of code @andela-jkariuki ?